src/Basho/Riak/Bucket.php

```diff
@@ -63,7 +67,9 @@ public function getName()
      * Returns the buckets R-value If it is set,
      * otherwise return the R-value for the client.
      *
-     * @return integer
+     * @param null $r
+     *
+     * @return int
      */
     public function getR($r = null)
```

Comment:
The only reason I can think why that's there is for testing purposes, still injecting the expected value of an accessor is pure evil. Also, unfortunately, it seems like that approach has been used a bit all around the places.
At first sight the accessor should return either the "object value" or the "client value" not a parameter for sure.